alloy/src/main/resources/META-INF/aui.tld

```diff
@@ -3,7 +3,7 @@
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-jsptaglibrary_2_1.xsd"
         version="2.1">
-   <tlib-version>3.0.5</tlib-version>
+   <tlib-version>3.0.6</tlib-version>
```

Comment:
Seems that the taglib2tld.xsl is generating the wrong version in this branch `3.0.x-legacy`.

This line is the issue:
https://github.com/liferay/liferay-faces/blob/master/alloy/src/main/resources/META-INF/facelets2tld.xsl#L11

@ngriffin7a, should I fix this or is it by-design?